.travis.yml

```diff
@@ -100,11 +95,6 @@ matrix:
            stage: Cron tests
            env: NUMPY_VERSION='dev' ASTROPY_VERSION='dev'
 
-         - python: 3.5
```

Comment:
I would like to keep running the figure tests against astropy dev, because changes to wcsaxes have bitten us in the past.